packages/react-i18n/src/index.tsx

```diff
@@ -12,18 +14,80 @@ export interface I18nReact {
 	_x: I18n[ '_x' ];
 	isRTL: I18n[ 'isRTL' ];
 	i18nLocale: string;
+	localeData?: LocaleData;
+	hasTranslation: ( singular: string, context?: string ) => boolean;
+	addFilter: typeof addFilter;
+	removeFilter: typeof removeFilter;
+	applyFilters: typeof applyFilters;
```

Comment:
Does `applyFilters` need to be returned from the `useI18n` hook and be part of public API? It looks more like internal private function used only by `bindI18nFunction`

Reply:
I thought it might be needed in some edge cases, but it wouldn't really have any real use case with the present needs in Calypso so it only aggravates unnecessarily the public API. Removed it in https://github.com/Automattic/wp-calypso/pull/44743/commits/e0f7d123711abfd6bda0541c977612e8035ab6b2